proposals/gc/MVP.md

```diff
@@ -123,20 +298,16 @@ In addition to the [existing rules](https://github.com/WebAssembly/function-refe
   - `i31 <: eq`
 
 * Any concrete type is a subtype of either `data` or `func`
-  - `(type $t) <: data`
+  - `$t <: data`
      - if `$t = <structtype>` or `$t = <arraytype>`
-     - or `$t = type ht` and `rt <: data` (imports)
-  - `(type $t) <: func`
+     - or `$t = type ht` and `ht <: data` (imports)
```

Comment:
Are these lines referring to type imports? I know they were already in here, but let's remove them since we don't plan to have type imports in the MVP.

Reply:
Right, removed dependence on type imports.